test/common/index.js

```diff
@@ -720,9 +720,8 @@ function expectsError(validator, exact) {
       assert.fail(`Expected one argument, got ${inspect(args)}`);
     }
     const error = args.pop();
-    const descriptor = Object.getOwnPropertyDescriptor(error, 'message');
     // The error message should be non-enumerable
-    assert.strictEqual(descriptor.enumerable, false);
+    assert.strictEqual(Object.prototype.propertyIsEnumerable.call(error, 'message'), false);
```

Comment:
Does this behaviour change? Isn't it a breaking change?

Reply:
If you insist I could revert this, now that i use super(msg) instead of getter and setters for message